app/src/common/css/mixins.scss

```
@@ -71,6 +71,84 @@
+    background-color: var(--rp-ui-base-bg-200);
+  }
+
+  svg {
+    width: 16px;
+    height: 16px;
+    flex-shrink: 0;
+    color: var(--rp-ui-base-e-400);
+    transition: color 0.2s ease-in-out;
+
+    path {
+      fill: currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)